### DISCUSSION TITLE
How can a new role include other role(s)? (aka inheritance)

### DISCUSSION THREAD

**Discussion by tolgaulas** [gravitational/teleport#13333] [Q&A]
Hi, bit lost in documentation. As I understood, you can not include other roles into a new role; for example if I already have a role "parent", then when I need new "role" child which should include parent role and something on top of it; how can it be done?

**Comment by webvictim** [comment]
This is correct, Teleport doesn't currently support role inheritance.

With Teleport, users get the permissions from every individual role that they're assigned. So if you have a user that needs both parent and child permissions, you can give them the parent role and create a separate child role with different permissions. They will get the full set of both.
